components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/persistence/beans/PersistenceConfigurations.java

```diff
@@ -19,13 +19,14 @@
 package org.wso2.carbon.stream.processor.core.persistence.beans;
 
 import org.wso2.carbon.config.annotation.Configuration;
+import org.wso2.carbon.stream.processor.core.persistence.FileSystemPersistenceStore;
 
 @Configuration(namespace = "state.persistence", description = "Query configurations for state persistence")
 public class PersistenceConfigurations {
     private boolean enabled = false;
     private int intervalInMin = 1;
     private int revisionsToKeep = 3;
-    private String persistenceStore = "org.wso2.carbon.stream.processor.core.persistence.FileSystemPersistenceStore";
+    private String persistenceStore = FileSystemPersistenceStore.class.getName();
     private PersistenceStoreConfigs config;
```

Comment:
If fileSystemPersistanceStore is not available in the pack this will throw class not found exception.

Reply:
Isn't it packed by default?

Reply:
Ya FileSystemPersistenceStore class is within stream.processor.core

Reply:
Ok since both are packed in the same jar it will be fine